bin/build

```diff
@@ -42,7 +44,8 @@ pushd $IMAGE_DIR
                     --build-arg APPLIANCE_REF=$APPLIANCE_REF \
                     --build-arg MIQ_ORG=$MIQ_ORG \
                     --build-arg SUI_ORG=$SUI_ORG \
-                    --build-arg APPLIANCE_ORG=$APPLIANCE_ORG"
+                    --build-arg APPLIANCE_ORG=$APPLIANCE_ORG \
+		    --build-arg ARCH=$ARCH"
```

Comment:
I'm also noticing some whitespace inconsistency here and elsewhere in the file (tabs here where the rest is spaces)...can you please fix?